src/Volume.jsx

```diff
@@ -2,27 +2,56 @@
 import { Button } from "@patternfly/react-core/dist/esm/components/Button";
 import { Checkbox } from "@patternfly/react-core/dist/esm/components/Checkbox";
 import { FormGroup } from "@patternfly/react-core/dist/esm/components/Form";
+import { FormHelper } from "cockpit-components-form-helper.jsx";
 import { FormSelect, FormSelectOption } from "@patternfly/react-core/dist/esm/components/FormSelect";
 import { Grid } from "@patternfly/react-core/dist/esm/layouts/Grid";
 import { TextInput } from "@patternfly/react-core/dist/esm/components/TextInput";
 import { TrashIcon } from '@patternfly/react-icons';
 import { FileAutoComplete } from 'cockpit-components-file-autocomplete.jsx';
 import cockpit from 'cockpit';
 
+import * as utils from './util.js';
+
 const _ = cockpit.gettext;
 
-export const Volume = ({ id, item, onChange, idx, removeitem, additem, options, itemCount }) =>
+export function validateVolume(volumes, key, validationFailed) {
+    const delta = { ...validationFailed };
+
+    switch (key) {
+    case "hostPath":
+        break;
+    case "containerPath":
+        if (utils.isEmpty(volumes.containerPath))
+            delta.containerPath = _("Container path must not be empty");
```

Comment:
These 6 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1354-20230907-093411-a9f37bd4-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)